ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -1,10 +1,33 @@
 #include "../PrjFSKext/kernel-header-wrappers/vnode.h"
 #import <XCTest/XCTest.h>
 #include "../PrjFSKext/KauthHandlerTestable.hpp"
+#include "../PrjFSKext/PerformanceTracing.hpp"
 #include "MockVnodeAndMount.hpp"
 
 using std::shared_ptr;
 
+#define KAUTH_RESULT_ALLOW    (1)
```

Comment:
These could also be enums.  Not sure what we gain/lose by one versus another.  If we do make it an enum note to self to initialize at 1.

Reply:
It doesn't really matter either way, except if we move it to a header, you might need to surround the definitions with `#ifdef KAUTH_RESULT_ALLOW` to avoid clashing with the kernel headers when building the testable kext.

Reply:
I switched them to enums so we don't have to worry about clashes